README.md

```diff
@@ -9,7 +9,7 @@
 
 ## Next major project milestone: Sharing with other funds
 
-Major new feature development on this project has more or less drawn to a close. If you are an abortion fund interested in whether this would work for you, hit us up and let us know!
+Major new feature development on this project has more or less drawn to a close. Our general administration/maintenance plan is [here](docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md). If you are an abortion fund interested in whether this would work for you, hit us up and let us know!
 
 ## [Come hang out with us on slack!](https://codefordc.slack.com/messages/dcaf_case_management)
 
```

Comment:
There's a line further down that say's we're a regular presence at CodeForDC meetups. Maybe we should redact that on this pull?

Reply:
Probably just easier to link to https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SECURITY.md

Reply:
great catch! Fixed in b199aaec

Reply:
On line 16, the Join via the Code for DC website link gets a 404 error